src/app/components/profile/profile.scss

```diff
@@ -0,0 +1,60 @@
+.box-container.custom-profile{
```

Comment:
another option could be extending .box-container and overriding specific attributes